playbooks/openshift-glusterfs/private/upgrade.yml

```diff
@@ -1,8 +1,36 @@
 ---
+- name: GlusterFS Upgrade Checkpoint Start
+  hosts: OSEv3
```

Comment:
Sorry, we should've provided more guidance, can you use oo_all_hosts group instead. Here and everywhere else you're using OSEv3?

Reply:
Done.